elements/rh-audio-player/rh-audio-player.ts

```diff
@@ -248,6 +248,14 @@ export class RhAudioPlayer extends LitElement {
 
   #dir = new DirController(this);
 
+  #width = this.offsetWidth;
```

Comment:
this might break in framework apps. do we have a test for instantiating with document.createElement?